src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBannerCircular.vue

```diff
@@ -2,135 +2,144 @@
 	<div>
 		<!-- open banner -->
 		<div
-			class="appeal-banner appeal-banner--open"
+			class="tw-py-2 lg:tw-py-3 tw-bg-brand-100"
 			v-if="isOpen"
 			key="openBanner"
 		>
-			<div class="row align-center align-middle relative">
-				<div class="appeal-banner__indicator shrink small-12 columns">
-					<div
-						class="indicator"
-						:class="isLoading ? 'indicator--is-loading' : ''"
-					>
-						<kv-progress-circle
-							class="indicator__progress-circle"
-							:stroke-width="12"
-							:value="goalPercent"
-							:arc-scale=".8"
-							:rotate="36"
-							:show-number="false"
-						/>
-						<kv-contentful-img
-							v-if="imageUrl"
-							class="indicator__image"
-							:contentful-src="imageUrl"
-							alt=""
-							fallback-format="gif"
-							:height="164"
-							:width="164"
-						/>
+			<kv-page-container>
+				<div class="tw-flex tw-flex-col tw-items-center tw-justify-center tw-relative">
+					<div class="tw-mb-4 lg:tw-m-0">
 						<div
-							class="indicator__goal-status"
-							v-html="goalStatus"
+							class="indicator tw-relative"
+							style="width: 10.25rem; height: 10.25rem;"
 						>
+							<kv-progress-circle
+								class="tw-absolute tw-h-full tw-w-full tw-z-2"
+								:stroke-width="12"
+								:value="goalPercent"
+								:arc-scale=".8"
+								:rotate="36"
+								:show-number="false"
+							/>
+							<kv-contentful-img
+								v-if="imageUrl"
+								class="indicator__image tw-absolute tw-h-full tw-w-full tw-z-1"
+								:class="isLoading ? 'tw-opacity-low' : ''"
+								:contentful-src="imageUrl"
+								alt=""
+								fallback-format="gif"
+								:height="164"
+								:width="164"
+							/>
+							<div class="tw-flex tw-justify-center tw-w-full tw-absolute tw-bottom-0">
+								<p
+									class="tw-w-10 tw-text-center tw-z-2 tw-font-medium tw-text-small
+										tw-my-0 tw-mx-[auto]"
```

Comment:
does `tw-mx-[auto]` need the brackets...

Reply:
Based on this comment https://github.com/kiva/ui/pull/3457#discussion_r759485473

But I think auto is a default value, so we can delete it @mcstover

Reply:
I think in that case he was referencing the `min` portion of the rule. But yea if `auto` is default we may be able to remove it. Just need to make sure `mx` isn't set to something else prior and needs to be reset.

Reply:
Is working fine without it and is fixed now.